cico_setup.sh

```diff
@@ -31,7 +31,8 @@ docker_login() {
 prep() {
     yum -y update
     yum -y install epel-release
-    yum -y install git which python36-pip python36-devel python36-virtualenv docker gcc
+    yum -y install git which python36-pip python36-devel docker gcc openssl-devel httpd-devel
+    pip3 install virtualenv pytest
```

Comment:
Is it good to install here? Or as part of runtests.sh ?

Reply:
the virtualenv is used in other scripts as well like linter, pydoc etc.